src/Core/Silk.NET.BuildTools/Cpp/Clang.cs

```diff
@@ -48,6 +48,12 @@ public static class Clang
             "LONG_PTR"
         };
 
+        public static bool IsProbablyABitmask(Enum @enum)
+            => @enum.Tokens.Count > 1 && // there is more than one token
```

Comment:
This condition doesn't need to exist?

Reply:
Not technically, but logically it does need to exist. An enum containing a single token of value 1 would be identified as a bitmask, but logically a bit mask requires more than one bit. 

A single token having just a bit enough is not enough certainty that we have a bitmask on our hands.